### PR TITLE
feat(config-cli): new command tool to bundle config json from a config path

### PR DESCRIPTION
**Pull request by Wentao-Kuang**

### CODE REVIEW

File: packages/config-cli/README.md [hunk 1]
```diff
@@ -23,3 +23,11 @@ Dump the screenshots with config file
 ./bin/bmc.js screenshot --config s3://..../config.json.gz
 
 ```
+
+## Usage -- Bundle
+
+Bundle config files into config bundle json from a given config path.
+
+```bash
+./bin/bmc.js screenshot --config config/ --bundle config/config.json
```
Comment (blacha) [inline packages/config-cli/README.md:32]: `bundle` ? 

bundling into the output directory could be interesting, maybe make the recommendation just `--output config.json` ?

Comment (Wentao-Kuang) [reply packages/config-cli/README.md]: Updated.

**Review by blacha — DISMISSED** [review]

File: packages/config-cli/src/cli/screenshot/screenshot.ts [hunk 2]
```diff
@@ -75,9 +75,12 @@ export class CommandScreenShot extends CommandLineAction {
 
     let BasemapsServer: FastifyInstance | undefined = undefined;
     if (config != null) {
-      const port = await getPort({ port: portNumbers(10000, 11000) });
+      const port = await getPort();
       host = `http://localhost:${port}`;
-      BasemapsServer = await startServer(host, port, config, logger);
+      // Force a default url base so WMTS requests know their relative url
+      process.env[Env.PublicUrlBase] = host;
```
Comment (blacha) [inline packages/config-cli/src/cli/screenshot/screenshot.ts:81]: if this is set we need to respect it we should not be overwriting env vars.

Comment (Wentao-Kuang) [reply packages/config-cli/src/cli/screenshot/screenshot.ts:81]: We can set this back after the script running in finally.

Comment (blacha) [reply packages/config-cli/src/cli/screenshot/screenshot.ts:81]: Proab safer to just use it if it exists the same as basemaps-server cli

Comment (Wentao-Kuang) [reply packages/config-cli/src/cli/screenshot/screenshot.ts:81]: Cool, updated.

**Pull request merged**